vendor/bat-native-ads/src/bat/ads/internal/classification/purchase_intent_classifier/purchase_intent_classifier.h

```diff
@@ -66,13 +87,15 @@ class PurchaseIntentClassifier {
       const std::vector<std::string>& set_b);
 
   bool is_initialized_;
-  uint16_t version_;
-  uint16_t signal_level_;
-  uint16_t classification_threshold_;
-  uint64_t signal_decay_time_window_in_seconds_;
+  uint16_t version_ = 0;
+  uint16_t signal_level_ = 0;
+  uint16_t classification_threshold_ = 0;
+  uint64_t signal_decay_time_window_in_seconds_ = 0;
   std::vector<SiteInfo> sites_;  // sites2segments
```

Comment:
I think I've left those comments in to reference the JSON attributes that come from the user-model-installer. Feel free to remove

Reply:
Thanks, I have now removed 👍